app/views/hyrax/admin/collection_types/_form_settings.html.erb

```diff
@@ -13,7 +13,7 @@
         <div class="checkbox">
           <input value="0" name="collection_type[share_applies_to_new_works]" type="hidden" <%= "disabled=\"disabled\"".html_safe if f.object.share_options_disabled? %> >
           <label id="sharable-applies-to-new-works-setting-label" class="boolean optional <%= 'disabled' if f.object.share_options_disabled? %>" for="collection_type_share_applies_to_new_works">
-            <input id="collection_type_share_applies_to_new_works" class="boolean optional <%= 'disabled' if f.object.share_options_disabled? %>" value="1" checked="checked" name="collection_type[share_applies_to_new_works]" type="checkbox" <%= "disabled=\"disabled\"".html_safe if f.object.share_options_disabled? %> >
+            <input id="collection_type_share_applies_to_new_works" class="boolean optional <%= 'disabled' if f.object.share_options_disabled? %>" value="1" <%= "checked='checked'" if f.object.share_applies_to_new_works %>" name="collection_type[share_applies_to_new_works]" type="checkbox" <%= "disabled=\"disabled\"".html_safe if f.object.share_options_disabled? %> >
```

Comment:
Is `.html_safe` in `"disabled=\"disabled\"".html_safe` necessary?   

And is `disabled` meant to be a class as well as an attribute on the element?

Maybe `"disabled=\"disabled\""` and `"checked='checked'"` are formatted the same way for consistency?  Generally you want to use double quotes in attribute values for HTML markup.

The logic looks good.  For readability sometimes it's current practice in modern code formatting tools to drop attributes to their own lines when a line gets long...  for example:

```
<input id="collection_type_share_applies_to_new_works" 
   class="boolean optional <%= 'disabled' if f.object.share_options_disabled? %>" 
   value="1" 
   <%= "checked='checked'" if f.object.share_applies_to_new_works %>" 
   name="collection_type[share_applies_to_new_works]" 
   type="checkbox" 
   <%= "disabled=\"disabled\"".html_safe if f.object.share_options_disabled? %> >
```

Reply:
The html basically mirrors that generated by simple_form for the `<%- f.input... %>` statements also on in this file.
* removed .html_safe on string constants
* disabled needs to be an attribute to make the element disabled. disabled needs to be a class to get the grayed out style.
* checked="checked" is fixed to have recommended quotes
* added line returns for readability

Reply:
I had to add .html_safe back in for html to be rendered correctly.

```
"disabled=\"disabled\"".html_safe    # disabled="disabled"
"disabled=\"disabled\""              # disabled=&quot;disabled&quot;
```